libtelemetry/build.gradle

```diff
@@ -39,6 +38,7 @@ dependencies {
     implementation project(':libcore')
     implementation dependenciesList.okhttp3
     implementation dependenciesList.gson
+    implementation dependenciesList.supportAnnotation
```

Comment:
this change was require due to the build issue shown in this CI job: https://circleci.com/gh/mapbox/mapbox-events-android/3022?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link